tests/testthat/test-tutorial.R

```diff
@@ -2,8 +2,8 @@ test_that("use_tutorial() checks its inputs", {
   skip_if_not_installed("rmarkdown")
 
   create_local_package()
-  expect_error(use_tutorial(), "no default")
-  expect_error(use_tutorial(name = "tutorial-file"), "no default")
+  expect_error(use_tutorial(), "must be a valid name")
+  expect_error(use_tutorial(name = "tutorial-file"), "must be a valid name")
```

Comment:
Would also convert these to snapshots.

(Probably all 3 of these are far from being "load-bearing" expectation and could also be considered for deletion. But let's err on the side up updating over deleting.)